saleor/graphql/app/types.py

```diff
@@ -597,6 +609,16 @@ def resolve_brand(root: models.App, _info: ResolveInfo):
             return root
         return None
 
+    @staticmethod
+    def resolve_breaker_state(root: models.App, _info: ResolveInfo):
+        if not breaker_board:
+            return CircuitBreakerState.CLOSED
```

Comment:
The field is nullable so maybe when no breaker is used, it should return null to indicate that?

Reply:
I could do that, but I'm not sure we want to expose configuration details like this through the API?

Reply:
I'm just wondering if it's not confusing when no breaker is used and the field will return one of the breaker's states. But you can also look at this from this perspective - when there is no breaker, the state is always "closed". But then the field shouldn't be nullable.